sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -26,7 +27,7 @@ object ScalaNativePluginInternal {
   val nativeWorkdir =
     taskKey[File]("Working directory for intermediate build files.")
 
-  val nativeConfig =
+  val nativeBuildConfig =
```

Comment:
This can probably be removed now, either by inlining its computation in `nativeConfig in key`, or in `nativeLink`, depending on what seems nicer (probably the former).